config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -301,8 +301,98 @@ presubmits:
           value: '{"api/beta":"true", "api/ga":"true"}'
         - name: LABEL_FILTER
           value: "Feature: isSubsetOf OffByDefault && !Alpha && !Deprecated && !Slow && !Disruptive && !Flaky"
-        - name: SKIP
-          value: PodSecurityPolicy|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|Network.should.set.TCP.CLOSE_WAIT.timeout|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|should.provide.basic.identity
```

Comment:
I think these skips are mostly (hopefully completely!) historic cruft that we shouldn't need anymore.

Let's try to remove this. Needs to be tested in a trial PR.

Reply:
if you conflate both changes and turn out to be needed we need to revert the entire commit. should not be better to separate the skip change and adding the new jobs?

Reply:
Okay, let's clean up the SKIP first: https://github.com/kubernetes/test-infra/pull/35570